src/components/MAPI/workernodes/ClusterDetailWorkerNodes.tsx

```diff
@@ -471,12 +476,17 @@ const ClusterDetailWorkerNodes: React.FC<IClusterDetailWorkerNodesProps> =
                     provider
                   )}
                 />
-                <ModifyNodePoolGuide
-                  clusterNamespace={cluster.metadata.namespace!}
-                />
-                <DeleteNodePoolGuide
-                  clusterNamespace={cluster.metadata.namespace!}
-                />
+
+                {!isReadOnly && (
+                  <>
```

Comment:
I think the spacing between the CLI guides changed a bit here
<img width="1220" alt="Screen Shot 2021-09-28 at 20 28 15" src="https://user-images.githubusercontent.com/62935115/135151501-ae7de1a2-5770-4013-92e7-ee71a1b9a93d.png">

Reply:
Lol 😬 thank you